### PR TITLE
Add getAllAWSRoleTypes

### PR DESCRIPTION
**Pull request by rballard-cai**
- Add getAllAWSRoleTypes
- Deprecate getAWSRoleTypes and getNonServiceAWSRoleTypes
- Update jsdoc-to-markdown to version that uses a requizzle version that works with node 12
- Add unit test
- Update docs and dist

### CODE REVIEW

**Comment by pofallon** [comment]
Thanks for submitting this!  I see where `requizzle` was added but I don't see where it was used?  Do we still need this dev dependency?

File: test/test.js [hunk 1]
```diff
@@ -213,6 +213,66 @@ describe('alks.js', function() {
...
+        status: 200
+      })
+
+      const result = await alks.getAllAWSRoleTypes({
+        baseUrl: 'https://your.alks-host.com',
+        accessToken: 'abc123',
+        _fetch
+      })
+
+      console.log(result)
```
Comment (americk0) [inline test/test.js]: Looks like you missed a console.log while debugging

Comment (rballard-cai) [reply test/test.js]: Good catch, fixed

**Comment by rballard-cai** [comment]
@pofallon it was a nested dependency of jsdoc-to-markdown, I updated the jsdoc-to-markdown version that uses the latest requizzle and deleted the unnecessary requizzle dev dependency.

**Comment by americk0** [comment]
@pofallon when you get the chance could you also approve or request changes? The Github rules on this repo require a second approver

**Pull request merged**